### PR TITLE
[Beat] Add table model schema for face payment

### PR DESCRIPTION
**Pull request by sonadztux**
## Type of changes

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Enhancement
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behaviour?
There's no model schema for the face payment solution.

## What is the new behavior?
- Add table model schema for face payment
- Add migrator for face payment's model
- Access the [ERD here](https://dbdiagram.io/d/614d6c6b825b5b0146113ba7)

ERD Face Payment

Ada beberapa penyesuaian pada struct di golang dari ERD di atas:
- `limit_payment` jadi `minimum_payment`
- `haveTwin` jadi `have_twin`
- `isActive` jadi `is_active`
- `face_payment_transactions.updated_at` dihapus karena sepertinya tidak dibutuhkan--ga ada operasi update buat tabel ini, setiap POST buat record baru.

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: backend/models/face_payment.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	"time"
+)
+
+type FacePaymentAccount struct {
+	ID				uint 			`gorm:"primaryKey; autoIncrement" json:"id"`
+	SessionID    	string  		
+	Visitor      	Visitor 		`gorm:"foreignKey:SessionID"`
+	FullName		string
+	Phone			string 			`gorm:"uniqueIndex"`
+	Pin				string
```
Comment (lazuardyk) [inline backend/models/face_payment.go:13]: Pin ini nggak int aja?
Soalnya definisinya PIN itu Personal Identification Number dan validasinya nanti harus int juga
Wdyt?

Comment (sonadztux) [reply backend/models/face_payment.go:13]: Dibuat string karena ga bakal ada operasi matematika buat pin

Comment (lazuardyk) [reply backend/models/face_payment.go:13]: Kalo dari frontend nya "ABC123" gimana? validasinya lu buat terpisah berarti? bukan dari struct

Comment (sonadztux) [reply backend/models/face_payment.go:13]: Iya musti dijaga bareng harusnya dari frontend juga ada validasi IsNumber. A reference: https://stackoverflow.com/questions/14171253/best-sql-type-for-a-credit-card-password

Comment (lazuardyk) [reply backend/models/face_payment.go:13]: Oalah okey

**Review by lazuardyk — APPROVED** [review]

**Review by bagasteguhimani — APPROVED** [review]
Naisuu

**Pull request merged**